libs/coin-evm/src/getTransactionStatus.ts

```diff
@@ -303,10 +303,10 @@ type EditTransactionValidationIssues = Partial<
  * the transaction will be rejected by the network with a
  * "replacement transaction underpriced" error
  */
-const validateEditTransaction = ({
-  editType,
+export const validateEditTransaction = ({
   transaction,
   transactionToUpdate,
+  editType,
```

Comment:
I just moved the `editType` in the param list to match the type ordering below (since `editType` is optional).
It does not change anything since the function param is an object, order does not matter